spyder/plugins/editor/widgets/completion.py

```diff
@@ -170,7 +170,7 @@ def show_list(self, completion_list, position, automatic):
         # signal used for testing
         self.sig_show_completions.emit(completion_list)
 
-    def update_list(self, filter_text):
+    def update_list(self, current_word, new=True):
```

Comment:
@metalogical could you add more information in this docstring about `current_word` and `new` parameters?